kolibri/plugins/setup_wizard/assets/src/state/constants.js

```diff
@@ -0,0 +1,3 @@
+import presets from '../../../../../auth/constants/facility_configuration_presets.json';
```

Comment:
It's import once versus import every time a user needs the static information. Making a dynamic endpoint for entirely static data feels weirder to me.